Libs/Visualization/VTK/Widgets/Testing/Cpp/ctkVTKScalarsToColorsViewTest2.cpp

```diff
@@ -107,11 +107,11 @@ int ctkVTKScalarsToColorsViewTest2(int argc, char * argv [] )
   ctkVTKScalarsToColorsView view(0);
   // add transfer function item
   vtkPlot* ctfPlot = view.addColorTransferFunction(ctf);
-  view.chart()->SetPlotCorner(ctfPlot, 1);
+  view.xyChart()->SetPlotCorner(ctfPlot, 1);
```

Comment:
Why is it called xyChart and not chartXY? If we follow VTK naming conventions then the method name should be chartXY.

Reply:
I believe that yes, thanks to Q_INVOKABLE.

Reply:
I agree that having `xyChart` may feel more natural and is aligned with the convention adopted by Qt in the Qt Chart[^1] module.

[^1]: https://doc.qt.io/qt-5/qtcharts-module.html

That said, CTK provides a convenience wrapper around VTK, for sake of consistency with the VTK naming convention, I think we should then have the set/get functions and associated internal variables match the VTK class names.

```
ls -1 vtkCh* 
vtkChartBox.cxx
vtkChartBox.h
vtkChart.cxx
vtkChart.h
vtkChartHistogram2D.cxx
vtkChartHistogram2D.h
vtkChartLegend.cxx
vtkChartLegend.h
vtkChartMatrix.cxx
vtkChartMatrix.h
vtkChartParallelCoordinates.cxx
vtkChartParallelCoordinates.h
vtkChartPie.cxx
vtkChartPie.h
vtkChartXY.cxx
vtkChartXY.h
vtkChartXYZ.cxx
vtkChartXYZ.h
```